tests/output/ConsoleInteractionTest.py

```diff
@@ -49,6 +49,15 @@
 STR_PROJECT_WIDE = "Project wide:"
 
 
+class DummyFileProxy:
+
+    def __init__(self, lines):
+        self.lines = lines
```

Comment:
I think a better test would be to have a `create_file_proxy` function which does 

```
def create_file_proxy(lines):
    fp = FileProxy('/dev/null')
    fp.__lines = lines
    return fp
```
as that will test file proxy rigorously in it's usage in all these files